src/switch-component.tsx

```diff
@@ -121,7 +121,7 @@ const Switch: React.FC<PropsRender | PropsChildren> = (props) => {
     return (
         // When the switch is disabled, it shouldn't be focusable
         // eslint-disable-next-line jsx-a11y/interactive-supports-focus
-        <span
+        <div
```

Comment:
why this change? I'm afraid this change could be problematic

Reply:
You're right. Instead of using div, I added `display: inline-block` to the span